Dockerfile

```diff
@@ -3,4 +3,4 @@ WORKDIR /root/src
 COPY . /root/src
 RUN ./build.sh
 USER builder
-ENTRYPOINT ["/usr/bin/coreos-assembler"]
+ENTRYPOINT ["dumb-init", "/usr/bin/coreos-assembler"]
```

Comment:
would there be any benefit to using a full path to dumb-init like we were doing for `/usr/bin/coreos-assembler` ?

Reply:
Might as well be consistent, sure.